packages/@magic-ext/connect/src/index.ts

```diff
@@ -12,8 +12,11 @@ export class ConnectExtension extends Extension.Internal<'connect', any> {
     return this.request<boolean>(requestPayload);
   }
 
-  public requestUserInfo() {
-    const requestPayload = this.utils.createJsonRpcRequestPayload(MagicConnectPayloadMethod.RequestUserInfo);
+  public requestUserInfo(options?: { isResponseRequired?: boolean }) {
```

Comment:
@harryEth @hcote @dgerrellsMagic I changed the name of the parameter to `isResponseRequired` because I think semantically it makes more sense.